app/assets/stylesheets/_billables.scss

```diff
@@ -53,6 +53,26 @@
   }
 }
 
+.unbill {
+  display: inline-block;
```

Comment:
Properties should be ordered background, border, border-color, color, cursor, display, font, min-width, padding